.github/workflows/build.yml

```diff
@@ -13,10 +13,17 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 8
-      uses: actions/setup-java@v1
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
       with:
-        java-version: 1.8
+        java-version: 11
+        distribution: 'adopt'
 
-    - name: Build with Maven
-      run: mvn -B package --file pom.xml
+    - name: Build with Gradle
+      run: gradlew build
+
+    - name: Upload Artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: Multiverse-Core-SNAPSHOT
+        path: build/libs/Multiverse-Core-4.3.2-SNAPSHOT.jar
```

Comment:
The version `4.3.2-SNAPSHOT` probably shouldn't be hard coded

Reply:
oops, good spot